src/Kernel/String.extension.st

```diff
@@ -0,0 +1,8 @@
+Extension { #name : #String }
+
+{ #category : #'*Kernel' }
+String >> isUnclassified [
+	"Return true if I am the name of Uncliassified protocol"
```

Comment:
I'm not fan of this one.
Moving away from symbols and strings is good, and reifing concepts is great, but adding back random methods to string feels not nice.

Reply:
I don't like it either but I'm not sure we can do without it until we transition.. Or do you have another idea?

Reply:
A possibility: for clients that manipulate protocols either as string or Protocol (I assume only external clients, because internal one should only deal with Protocol instance?), you can transform the variable early with some `asProtocol` or something (no-op on Protocol but something on String)

Reply:
You prefer to have #asProtocol on String?

Reply:
yes, because `asXYX` has a clear traditional meaning of retuning the instance of XYZ, the one of the given name. eg `asPackage` or even `asClass`.

Whereas `isUnclassified` means nothing obvious on String instances. is `'foo'` classified? what is a classified string? is there digits or Chinese characters inside?
I naturally expect that predicates (testing protocol) on strings answer relatively to intrinsic property of the string content. e.g. `isAllDigits`

Moreover, the selector `isUnclassified` gives no indication it is related to protocol in any way.

Reply:
Note: in fact, I don't know if Protocol objects are global (one Protocol instance by name on the whole system, this feels bad) or local to a class (each class has its own collection of protocols, this feels better). For the latter, `asProtocol` does not work and you should ask the targeted class to do the conversion string->protocol